client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -114,14 +136,20 @@ const UserPage = createClass({
 								{`${(this.state.sortDir == 'asc' ? '\u25B2 ASC' : '\u25BC DESC')}`}
 							</button>
 						</td>
+						{this.renderFilterOption()}
 					</tr>
 				</tbody>
 			</table>
 		</div>;
 	},
 
 	getSortedBrews : function(){
-		return _.groupBy(this.props.brews, (brew)=>{
+		const testString = _.deburr(this.state.filterString).toLowerCase();
+		const brewCollection = this.state.filterString ? _.filter(this.props.brews, (brew)=>{
+			return (_.deburr(brew?.title).toLowerCase().includes(testString)) ||
+			(_.deburr(brew?.description).toLowerCase().includes(testString));
```

Comment:
I need to double check if this is safe for brews with no `brew.description`; of course I also realize now that optional chaining is not the correct way to handle that scenario.

Reply:
Perhaps something like
```
[...] ||
(brew.description && _.deburr(brew.description).toLowerCase().includes(testString))
```

Reply:
Remember, the `?.` syntax checks for the existence of the *preceding* element. The code above is checking Whether `brew` is undefined, not `brew.description`.

Reply:
Although - I'm not sure if this would work - simply moving the optional chaining one step further down the chain might be a solution:
```
_.deburr(brew.description)?.toLowerCase().includes(testString)
```

Reply:
Your earlier solution should definitely work. This one might also work as long as `_.deburr` can handle undefined values and returns something falsy.

Reply:
I'll run a couple of tests; I should be able to push through an update to the PR in the next couple of hours.

Reply:
I started by running tests with the suggested change, and it handles the absence of `brew.description` with no issues at all, so it appears that I was concerned over a scenario that doesn't even cause a problem.